INSTALL.adoc

```diff
@@ -43,6 +43,7 @@ Installation using `configure`
 
 Optionally, you can use the `configure` script to detect dependencies:
 
+	$ make configure	# if you're building from the Tig repository (requires `autoconf`)
```

Comment:
`./autogen.sh`

Reply:
Alright, thanks for the info @jtheoof. Should I change the line to `$ ./autogen.sh` or do nothing and just close this PR?

Reply:
No I would keep the MR but using my recommended fix.

Perhaps @jonas will merge. It's fairly easy to review.

Reply:
If this is what you've meant - I've changed it to `./autogen.sh`.